### DISCUSSION TITLE
Is there a way to define the default address space on Azure?

### DISCUSSION THREAD

**Discussion by jeffreyflynt** [edgelesssys/constellation#2883] [Q&A] [answered]
When creating a cluster on Azure, the default address space is: `10.0.0.0/8`; however, this overlaps with another address space in Azure. Is there a way to trim this down to say something like: `10.0.1.0/24`? 

I can't find any references to this upon initial config generation, only after it's created do I see the references.

**Comment by 3u13r** [comment, marked as answer]
Hello,
I assume you want to peer the two currently overlapping VPCs?
You get full control over the infrastructure when using our [terraform provider](https://docs.edgeless.systems/constellation/workflows/terraform-provider). You can find an example [main.tf here](https://github.com/edgelesssys/constellation/blob/main/terraform-provider-constellation/examples/full/azure/main.tf). Since the VPC CIDR is not exposed in our `azure_infrastructure` module, you need to download the .zip containing the infrastructure module unpack it in the same folder and point the module source to it. Then you have full control over all CIDRs and edit the VPC CIDR in https://github.com/edgelesssys/constellation/blob/main/terraform/infrastructure/azure/main.tf#L195. Note that the VPC contains multiple subnets, so if you want to use a /24 for the VPC you also need to decrease the subnet sizes.

I hope this helps. Please don't hesitate to ask if you encounter any problems along the way.
